### PR TITLE
document default handler behavior

### PR DESCRIPTION
**Pull request by agoddard**
Document the expectations & behavior surrounding the "default" handler

### CODE REVIEW

File: source/docs/0.23/handlers.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ next:
 
 - [What is a Sensu event handler?](#what-is-a-sensu-event-handler)
   - [Handler types](#handler-types)
+  - [Default Handler](#default-handler)
```
Comment (calebhailey) [inline source/docs/0.23/handlers.md]: s/Default Handler/The default handler/ 

...and the anchor link will need to be #the-default-handler

File: source/docs/0.23/handlers.md [hunk 2]
```diff
@@ -53,6 +54,19 @@ interact with almost any computer program via [standard streams][4].
   event handlers, making it easy to manage groups of actions that should be
   executed for certain types of events.
 
+### Default handler
```
Comment (calebhailey) [inline source/docs/0.23/handlers.md]: s/Default Handler/The default handler/

File: source/docs/0.23/handlers.md [hunk 3]
```diff
@@ -53,6 +54,19 @@ interact with almost any computer program via [standard streams][4].
...
+
+Sensu expects all [events][2] to have a corresponding handler. Event handler(s)
+may be configured in the [check definition][16], however if no `handler` or
```
Comment (calebhailey) [inline source/docs/0.23/handlers.md]: "may be configured in check definitions" (plural, and drop "the")?

**Comment by calebhailey** [comment]
Other than the few little edits I suggested above, this LGTM. 👍

**Comment by agoddard** [comment]
This should fix #98

**Comment by calebhailey** [comment]
Rebased this against the refactored docs so it can be merged. \o/

**Pull request merged**